blueprints/banking_blueprint/__init__.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+"""This module contains a MindMeld application"""
+from mindmeld import Application
+from mindmeld.models import entity_features
+from mindmeld.markup import process_markup
+from mindmeld.core import FormEntity
+
+app = Application(__name__)
+
+__all__ = ['app']
+
+
+sample_user = {
+    'savings' : 500,
+    'checking' : 100,
+    'credit' : 60,
+    'autoPay' : False,
+}
+
+
+@app.handle(intent='greet')
+def greet(request, responder):
+    replies = ['Hello there! Welcome to MindMeld Bank', 'Hello, thanks for choosing MindMeld Bank.']
+    responder.reply(replies)
+
+@app.handle(intent='lost_creditcard')
+def faq_lost_creditcard_handler(request, responder):
+    replies = ['If your card is lost or stolen, or you think someone used your account without permission, tell us immediately by calling 1-800-432-3117.']
+    responder.reply(replies)
+
+@app.handle(intent='new_creditcard')
+def faq_new_creditcard_handler(request, responder):
+    replies = ['In order to open a new credit card you must visit our website MindMeldBank.com or visit a local branch.']
+    responder.reply(replies)
+
+@app.handle(intent='order_checks')
+def faq_order_checks_handler(request, responder):
+    replies = ['We have placed an order for a checkbook. To confirm, change quanity of checks, or any other questions please view confirmation email.']
+    responder.reply(replies)
+
+@app.handle(intent='routing_number')
+def faq_routing_number_handler(request, responder):
+    replies = ['Your routing number is 121122676.']
```

Comment:
I think this information can be mapped to the customer data and fetched from there.